### DISCUSSION TITLE
v5.0.0alpha7

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#15789] [General]
All notable changes to this version has been documented in the [CHANGELOG-5.0.md](https://github.com/phalcon/cphalcon/blob/master/CHANGELOG-5.0.md) file.

Supported PHP versions: 7.4, 8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/v5.0.0alpha7'>v5.0.0alpha7</a>.</em>

**Comment by borisdelev** [comment]
Thanks :)))

A question... in documentation is mention that ```Removed Phalcon\Helper\Str - replaced by Phalcon\Support\Helper\Str\*``` so...
IF i try : ```\Phalcon\Support\Helper\Str::camelize('bobi');``` did not work (PHP8, Win: error is Uncaught Error: Class "Phalcon\Support\Helper\Str" not found)... where i am wrong? 

PS: is phalcon/ide-stubs updated to alpha7?

**Reply by niden** [reply]
Stubs: not yet. We will get on it shortly. I believe there were a couple of issues that we needed to address first.
Camelize: This was an omission that has been already addressed. Have a look at this issue and a workaround for now until the next release

https://github.com/phalcon/cphalcon/issues/15807

**Reply by niden** [reply]
Also note that the classes are no longer static. They have to be instantiated. The easiest way for you to do this is to use the `helper` service from the DI which is a factory creating all these classes for you.

```php
$this->helper->camelize('bobi');
```

or if you want to do it directly

```php
<?php

use Phalcon\Support\Helper\Str\Camelize;

$camelize = new Camelize();

$text = $camelize('bobi');  // or $camelize->__invoke('bobi');
```

**Reply by borisdelev** [reply]
You are great! Thank a lot and... may the force be with you :)
